### PR TITLE
Add boutdata

### PR DESCRIPTION
**Pull request by johnomotani**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by johnomotani** [comment]
@ZedThree, @loeiten, please comment to confirm you're happy to be maintainers.

File: recipes/boutdata/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+test:
+  imports:
+    - boutdata
+  requires:
+    - pytest
+  source_files:
+    - test
+  commands:
+    - pytest
```
Comment (marcelotrevisani) [inline recipes/boutdata/meta.yaml:38]:
```suggestion
```

Comment (johnomotani) [reply recipes/boutdata/meta.yaml:38]: Why do you suggest removing the `pytest` test @marcelotrevisani?

**Comment by ZedThree** [comment]
Yes, happy to be a maintainer!

**Comment by loeiten** [comment]
I confirm that I'm happy to be a maintainer.

File: recipes/boutdata/meta.yaml (hunk 1 above)
Comment (johnomotani) [reply recipes/boutdata/meta.yaml:38]: @marcelotrevisani sorry I didn't understand your suggestion. Is the change needed before merging? Thanks!

**Comment by johnomotani** [comment]
@conda-forge/staged-recipes I think this is ready to merge. Thanks!

**Pull request merged**